src/base.jl

```diff
@@ -27,24 +27,21 @@ A power system defined by fields for base_power, components, and forecasts.
 
 ```julia
 System(base_power)
-System(components, base_power)
-System(buses, generators, loads, branches, storage, base_power, services, annex; kwargs...)
-System(buses, generators, loads, base_power; kwargs...)
+System(base_power, buses, components...)
+System(base_power, buses, generators, loads, branches, storage, services; annex, kwargs...)
+System(base_power, buses, generators, loads; kwargs...)
 System(file; kwargs...)
 System(; buses, generators, loads, branches, storage, base_power, services, annex, kwargs...)
 System(; kwargs...)
 ```
 
 # Arguments
-- `buses::Vector{Bus}`: an array of buses
-- `generators::Vector{Generator}`: an array of generators of (possibly) different types
-- `loads::Vector{ElectricLoad}`: an array of load specifications that includes timing of the loads
-- `branches::Union{Nothing, Vector{Branch}}`: an array of branches; may be `nothing`
-- `storage::Union{Nothing, Vector{Storage}}`: an array of storage devices; may be `nothing`
 - `base_power::Float64`: the base power value for the system
-- `services::Union{Nothing, Vector{<:Service}}`: an array of services; may be `nothing`
+- `buses::Vector{Bus}`: an array of buses
+- `components...`: Each element must be an iterable containing subtypes of Component.
 
 # Keyword arguments
+- `annex::Dict`: Defines LoadZones if key :LoadZone is present
```

Comment:
should we change this to `ext` for consistency with our components?

Reply:
> should we change this to `ext` for consistency with our components?

if so, we can do so in a separate PR

Reply:
We should also remove the comment since now we have a topology object for this

Reply:
I don't think I saw it being used.  Not if any other script is using it.  Could we remove the option altogether? The user can pass LoadZone object in `components...`.

Reply:
I think we should keep the option of adding ext.

Reply:
We already have `ext` for `System`. Two questions:
1. Do you want to allow the user to pass it here?  If you say yes then we should probably make sure all of the constructors support that.  I could do that here.
2. I think it would be better to remove LoadZone handling here.  That's contingent on whether we can break backwards compatibility. Thoughts?

Reply:
I don't think anyone is creating LoadZones like that

Reply:
I think it's ok to break bw compatibility for this.